fileconnector/fileconnector-1.0.0/org.wso2.carbon.connector/src/main/java/org/wso2/carbon/connector/FileDelete.java

```diff
@@ -66,7 +66,7 @@ public void connect(MessageContext messageContext) throws ConnectException {
 		try {
 			resultStatus = deleteFile(fileLocation, filename, filebeforepprocess);
 		} catch (FileSystemException e) {
-			generateResults(messageContext, resultStatus);
+			handleException("Error while Deleting a file/folder", e, messageContext);
```

Comment:
Found System.out.println("File deletion started..."); Maybe this is from a previous commit. Better remove it.